resources/dicts/events/misc_events/general/medicine.json

```diff
@@ -17,4 +17,23 @@
         "other_cat_trait": null,
         "other_cat_skill": null
     }
-]
\ No newline at end of file
+],
+  {
+        "camp": "any",
+        "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "other_clan"],
+        "event_text": "m_c visits o_c to converse with their medicine cats about an omen StarClan sent.",
+        "cat_trait": ["strong connection to starclan", "omen sight", "clairvoyant", "dream walker"],
+        "cat_skill": ["compassionate", "faithful", "altruistic"],
+        "other_cat_trait": null,
+        "other_cat_skill": null
+    },
+{
+        "camp": "any",
+        "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "other_clan"],
+        "event_text": "m_c visits o_c to congratulate them on their new medicine cat."],
```

Comment:
there's some weird stuff going on with your brackets here. this one in line 33 is unneeded, and some of the curly braces should be [] instead

Reply:
@Ryos00 i've changed some of the brackets and i hope i've gotten it right? i put the bracket on the wrong line for line 33, was meant to go into 34. which curly brackets do i need to change?